CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Pundit Matchers
 
+## 3.0.1 (unreleased)
+
+- Allow `permit_action` to match dynamic actions
```

Comment:
This will also allow `forbid_action`, `permit_actions`, and `forbid_actions`. @chrisalley do you have a better suggestion? Should `permit_action` just be omitted?

Reply:
I would think that the four matchers should behave in a consistent way.

Reply:
Sorry, I meant that this fix applies to all the matchers not just `permit_actions`, so this can be rephrased as

```
- Fix a regression that prevented to match dynamically defined actions
```

```
- Allow to match dynamically defined actions

```
- Allow to match dynamic actions
```

or something like this